### PR TITLE
fix: typos in French dialogues

### PR DESCRIPTION
**Pull request by wartab**

### CODE REVIEW

**Comment by Adri1** [comment]
Thanks for your review

However, "sure" and "arrachepied" are not typos, but orthographical rectifications of 1990

https://fr.wiktionary.org/wiki/sure
https://fr.wiktionary.org/wiki/d%E2%80%99arrachepied

File: src/locales/fr/dialogue.ts [hunk 1]
```diff
@@ -2233,21 +2233,21 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+				$@c{neutral_eclosed}D’ailleurs, qui peut dire aujourd’hui qu’ils ont pu être réels ?
 				$@c{serious_mopen_fists}Il faut que tu persévères. Si tu t’arrêtes, ça n'aura jamais de fin et t’es la seule à en être capable.
-				$@c{serious_smile_fists}Difficile connaitre le sens de tout ça, je sais juste que c’est la réalité.
+				$@c{serious_smile_fists}Difficile de comprendre le sens de tout ça, je sais juste que c’est la réalité.
 				$@c{serious_mopen_fists}Si tu ne parviens à pas me battre ici et maintenant, tu n’as aucune chance.`
     },
     "victory": {
       1: `@c{smile_eclosed}J'ai fait ce que je j'avais à faire.
-				$Prmoets-moi juste une chose.\n@c{smile}Après avoir réparé ce monde… Rentre à la maison.`
+				Prmets-moi juste une chose.\n@c{smile}Après avoir réparé ce monde… Rentre à la maison.`
```
Comment (XGSleepWalker) [inline src/locales/fr/dialogue.ts:2250]: This is still incorrect.

File: src/locales/fr/dialogue.ts [hunk 2]
```diff
@@ -2094,16 +2094,16 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
     },
   },
   "rival_female": {
     "encounter": {
       1: `@c{smile_wave}Ah, je te cherchais ! Je t’ai cherché partout !\n@c{angry_mopen}On oublie de dire au revoir à sa meilleure amie ?
                     $@c{smile_ehalf}T’as décidé de réaliser ton rêve, hein ?\nCe jour est donc vraiment arrivé…
-					$@c{smile}Je veux bien te pardonner de m’avoir oubliée, à une conditon. @c{smile_wave_wink}Que tu m’affronte !
-					$@c{angry_mopen}Donne tout ! Ce serait dommage que ton aventure finisse avant d’avoir commencé, hein ?`
+					$@c{smile}Je veux bien te pardonner de m’avoir oubliée, à une conditon. @c{smile_wave_wink}Que tu m’affrontes !
+					$@c{angry_mopen}Donne tout ! Ce serait dommage que ton aventure finisse avant d’avoir commencée, hein ?`
```
Comment (XGSleepWalker) [inline src/locales/fr/dialogue.ts:2106]: Wasn't this one correct in the first place?

File: src/locales/fr/dialogue.ts
Comment (XGSleepWalker) [inline src/locales/fr/dialogue.ts:1]: Please review the whole file once more, there are a couple of additional lines incorrect. 

Examples: 11: "J'ai tout donné pour élever mes Pokémon. Attention à toi si tu leur fait du mal !",

**Review by XGSleepWalker — CHANGES_REQUESTED** [review]

**Comment by Adri1** [comment]
All the real typo corrections have been integrated into PR 1433 I open about typos before, with some extra.

**This PR can be closed**

**Comment by XGSleepWalker** [comment]
There are conflicts to be resolved. Once those are cleared, I'll approve it.

**Comment by XGSleepWalker** [comment]
Closed since these changes were included in https://github.com/pagefaultgames/pokerogue/pull/1433

File: src/locales/fr/dialogue.ts (hunk 2 above)
Comment (EdgarZimmer) [reply src/locales/fr/dialogue.ts]: "affronte" was incorrect, though "avoir commencé" was correct

**Pull request closed without merging**